File: mobile_through_project_final/src/main/java/com/ssafy/cafe/model/dto/Coupon.java
package com.ssafy.cafe.model.dto;

import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class Coupon {
    private Integer id;
    private Integer typeId;
    private String userId;
    private Boolean isUse;
    
    @Builder
    public Coupon(Integer id, Integer typeId, String userId, Boolean isUse) {
        super();
        this.id = id;
        this.typeId = typeId;
        this.userId = userId;
        this.isUse = isUse;
    }
}

File: app/src/main/java/com/ssafy/cobaltcoffee/dto/Coupon.kt
package com.ssafy.cobaltcoffee.dto

import com.google.gson.annotations.SerializedName

data class Coupon(
    val id: Int,
    val typeId: Int,
    val userId: String,
    @SerializedName("use") val isUse: Boolean
) {
    constructor() : this(0, 0, "", false)
}